R/sample_and_adjust.R

```diff
@@ -32,7 +32,7 @@
 #' @param verbose verbose output?
 #' @returns a named list containing `final_regimen` (all doses after
 #' adjustment), `tdms` (all collected levels, both true and measured, that is,
-#' both with and without residual variability), and `additional_info`, which
+#' both with and without residual variability), and `est_parameters`, which
```

Comment:
renaming this output variable, as it was just holding the fitted parameters.

Reply:
in another PR we add other info here for other fit methods. i would prefer to keep this generic.

Reply:
ah didn't see that. I can move it back in. My thinking was that "additional_info" wasn't really something where a user would think to look for parameters, so created a separate object to collect tghat. I would still leave it separate. IMO we should leave `additional_info` to collect only more miscellaneous stuff.